x-pack/plugins/fleet/server/types/rest_spec/epm.ts

```diff
@@ -247,3 +247,13 @@ export const DeletePackageRequestSchemaDeprecated = {
     })
   ),
 };
+
+export const GetInputsRequestSchema = {
+  params: schema.object({
+    pkgName: schema.string(),
+    pkgVersion: schema.string(),
+  }),
+  query: schema.object({
+    format: schema.oneOf([schema.literal('json'), schema.literal('yml'), schema.literal('yaml')]),
```

Comment:
We can add a `default` of JSON here to fallback to JSON on an unknown or unspecified format query param